### PR TITLE
pxl_scripts/px/http_data_terse: adding new script for http data w/ ve…

### PR DESCRIPTION
**Pull request by etep**
http_data_terse:
Similar to http_data, but with a terse display. Targeted at the "quick start guide" so that a new user has a very simple script to run, with a very terse display.

### CODE REVIEW

File: pxl_scripts/px/http_data_terse/data.pxl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Script variables
+# ----------------------------------------------------------------
+start_time = '-30s'
+max_num_records = 10
+
+# ----------------------------------------------------------------
+# Implementation
+# ----------------------------------------------------------------
+
+#f = px.DataFrame(table='http_events', start_time=start_time)
```
Comment (zasgar) [inline pxl_scripts/px/http_data_terse/data.pxl:22]: rm

Comment (etep) [reply pxl_scripts/px/http_data_terse/data.pxl]: removed that stray LOC. good catch.

Comment (oazizi000) [inline pxl_scripts/px/http_data_terse/data.pxl]: rm

**Review by oazizi000 — APPROVED** [review]
Long-term we have to find a way to deduplicate common scripts, probably by using parameters.

But we want this for the quick-start guide, so good to go.

**Review by oazizi000 — APPROVED** [review]

**Review by zasgar — APPROVED** [review]

**Pull request merged**